WordPress/Classes/Services/BlogService.m

```diff
@@ -941,19 +943,21 @@ - (BlogDetailsHandler)blogDetailsHandlerWithBlogObjectID:(NSManagedObjectID *)bl
                                        completionHandler:(void (^)(void))completion
 {
     return ^void(RemoteBlog *remoteBlog) {
-        [self.managedObjectContext performBlock:^{
-            NSError *error = nil;
-            Blog *blog = (Blog *)[self.managedObjectContext existingObjectWithID:blogObjectID
-                                                                           error:&error];
-            if (blog) {
-                [self updateBlog:blog withRemoteBlog:remoteBlog];
+        [[[JetpackCapabilitiesService alloc] init] syncWithBlogs:[NSArray arrayWithObject:remoteBlog] success:^(NSArray<RemoteBlog *> *blogs) {
```

Comment:
You can use the `@[]` shorthand to create an array

```suggestion
        [[[JetpackCapabilitiesService alloc] init] syncWithBlogs:@[remoteBlog] success:^(NSArray<RemoteBlog *> *blogs) {
```

Reply:
👀 

0918438ae8333ba13cf325ab12648e762b7bb81c